portfolio/src/main/webapp/script.js

```diff
@@ -73,4 +73,10 @@ function addRandomProject() {
   linkElement = document.getElementById('link-container');
   linkElement.text = link.description;
   linkElement.href = link.url;
-}
\ No newline at end of file
+}
+
+function appendData() {
```

Comment:
Should the Functions in this file be in alphabetical order? In general when should things be in alphabetical order in code? Does it usually matter?